cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/elasticsearch/ElasticsearchBackendImpl.java

```diff
@@ -25,9 +25,9 @@
 import com.telefonica.iot.cygnus.errors.CygnusPersistenceError;
```

Comment:
An entry to CHANGES_NEXT_RELEASE file, briefly describing the modifications in this PR and referencing the issue number, should be included. Please have a look to existing CHANGES_NEXT_RELEASE file to see the style we use.

Reply:
Fixed in 5101c07